### DISCUSSION TITLE
Marks file

### DISCUSSION THREAD

**Discussion by azinsharaf** [gokcehan/lf#1319] [Q&A] [answered]
hi, Is there any sample file for Marks I can use? I need to know its syntax.

**Comment by joelim-work** [comment, marked as answer]
You can just try creating some marks by pressing `m` (e.g. `ma` creates a mark called 'a' at the current location), and then viewing the marks file afterwards. The location of the marks file is as shown below:

https://github.com/gokcehan/lf/blob/e885abe1a3f6a0a6a795f7f75f601c80566a3de2/doc.go#L275-L278

For reference, here is an example marks file that I created just now:

```
a:C:\Users\joe.lim\Documents
b:C:\Users\joe.lim\Downloads
z:C:\Program Files
```
